scripts/build.sh

```diff
@@ -72,16 +68,33 @@ function check_dependencies_version() {
     echo "GraphScope requires python 3.6 or greater."
     exit 1
   fi
-  if ! command -v cargo &> /dev/null; then
+  if [[ "${platform}" != *"Darwin"* ]]; then
```

Comment:
Why mac doesn't need the maven

Reply:
and cargo

Reply:
and go

Reply:
since gie is not ready yet in mac. Will remove these skip when gie ready.